packages/side-nav/src/SideNavGroup.tsx

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { css, cx } from '@leafygreen-ui/emotion';
+import { uiColors } from '@leafygreen-ui/palette';
+import { ulStyleOverrides, LEFT_RIGHT_OFFSET } from './styles';
+
+const sideNavLabelStyle = css`
```

Comment:
If we already have styles split from the component, should this class live in the styles file instead?

Reply:
I only split out common styles (anything reused). Whenever possible otherwise, I try to keep styles alongside components.

Reply:
That's fair enough!